packages/jellyfish-api-core/__tests__/category/loan/createLoanScheme.test.ts

```diff
@@ -0,0 +1,108 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { UTXO } from '@defichain/jellyfish-api-core/category/oracle'
+import BigNumber from 'bignumber.js'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+
+    // NOTE(jingyi2811): Default scheme
+    await client.loan.createLoanScheme(100, new BigNumber(1.5), { id: 'default' })
+    await container.generate(1)
+  })
+
+  afterEach(async () => {
```

Comment:
This seems a bit overkill as I create 1 non-default scheme only in all the test units. 
However, I plan to add this afterEach function to all scheme related rpcs and I think it is good to standardize the code .